Library/Formula/sstp-client.rb

```diff
@@ -0,0 +1,33 @@
+class SstpClient < Formula
+  homepage "http://sstp-client.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/sstp-client/sstp-client/1.0.9/sstp-client-1.0.9.tar.gz"
+  sha1 "b56c60fc6ecd8b1686d6c319a8adb3f988bcabbd"
+
+  depends_on "libevent"
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--disable-ppp-plugin",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+
+    # Create a directory needed by sstpc for privilege separation
+    mkdir_p "var/run/sstpc"
+  end
+
+  def caveats; <<-EOS.undent
+    sstpc reads PPP configuration options from /etc/ppp/options. If this file
+    does not exist yet, type the following command to create it:
+
+    sudo touch /etc/ppp/options
```

Comment:
I took a closer look and it is not `sstp-client` that looks for `/etc/ppp/options` - `sstp-client` seems to invoke `pppd` (which is shipped with OS X) and then `pppd` looks for its own config file in `/etc/ppp/options` - which does not seem to exist by default. You can confirm this by simply running `sudo pppd` from the command line - on my machine it simply complains about not being able to open `/etc/ppp/options`.